src/api/store/event.py

```diff
@@ -893,6 +893,26 @@ def rsvp_remove(self, context: Context, args) -> Tuple[dict, MassEnergizeAPIErro
         raise Exception("events.rsvp.remove: must specify rsvp or event id")
           
       return result, None
+    except Exception as e:
+      capture_message(str(e), level="error")
+      return None, CustomMassenergizeError(e)
+    
+
+
+  def share_event(self, context: Context, args) -> Tuple[dict, MassEnergizeAPIError]:
+    try:
+      event_id = args.pop("event_id", None)
+      event = Event.objects.filter(id=event_id).first()
+      shared_to = args.pop("shared_to", None)
+      if shared_to:
+        first = shared_to[0]
+        if first == "reset": 
```

Comment:
There should be a way for a community admin to unshare just to their community or communities, not to all communities.  It looks like this would unshare to all communities which if a cadmin shouldn't be able to do.